webserver.py

```diff
@@ -72,8 +72,8 @@ def initialize_state(game_state):
         for j in range(GAME_CONFIG['field_height']):
             if game_state["plot" + str(i) + "," + str(j)]['seedType'] != 0:
                total_crops_planted += 1
-    if total_seed_count == 0 and total_crops_planted == 0 and game_state['resources']['cash'] < 5:
-        game_state['resources']['cash'] = 5
+    if total_seed_count == 0 and total_crops_planted == 0 and game_state['resources']['cash'] < GAME_CONFIG['starting_resources'][resource]:
```

Comment:
I don't think this change is what you want.  `resource` will just happen to be whatever value it was at the end of the loop above.